specs/decimal/decimal.md

```diff
@@ -45,7 +57,152 @@ data structure may be re-expressed in each language using language-specific cons
 defines the `DecimalValue` structure as the Win32 `DECIMAL` definition syntax is incompatible with
 WinRT IDL.
 
-# 3. WinRT API
+# 3. Examples
```

Comment:
It would be helpful to have examples of an API that takes/returns a Decimal type. Like a CurrencyTextBox control that has a Decimal-typed Amount property.